rest_framework/renderers.py

```diff
@@ -1053,6 +1053,8 @@ def __init__(self):
         assert yaml, 'Using OpenAPIRenderer, but `pyyaml` is not installed.'
 
     def render(self, data, media_type=None, renderer_context=None):
+        # disable yaml advanced feature 'alias' for clean, portable, and readable output
+        yaml.Dumper.ignore_aliases = lambda *args: True
```

Comment:
Is it possible to do this without overriding `ignore_aliases` on the `Dumper` class? I'm wondering what the interaction is for anyone else who wishes to use `yaml.dump` after this call.